src/biogeochem/CNPhenologyMod.F90

```diff
@@ -3035,19 +3022,25 @@ subroutine CNLitterToColumn (bounds, num_soilc, filter_soilc,         &
                         if (.not. use_grainproduct) then
                          ! grain litter carbon fluxes
                          phenology_c_to_litr_met_c(c,j) = phenology_c_to_litr_met_c(c,j) &
-                              + grainc_to_food(p) * lf_flab(ivt(p)) * wtcol(p) * leaf_prof(p,j)
+                              + grainc_to_food(p) * lf_flab(ivt(p)) * wtcol(p) * leaf_prof(p,j) &
+                              + biofuelc_harvest(p) * lf_flab(ivt(p)) * wtcol(p) * leaf_prof(p,j)  ! biofuel
```

Comment:
I wonder if we want to add a different switch for biofuel harvest, rather than lumping it with grain product harvest. Perhaps not, but worth at least thinking about. The code in this if (.not. use_grainproduct) loop will put the grain (and as currently written) the biofuel harvest back into the litter pool. Should we separate them so that we can choose to turn off one (either grain or biofuel residue) without impacting the other?